syfertext/tokenizer.py

```diff
@@ -140,8 +140,9 @@ def __call__(self, text: Union[String, str]):
         # this tokenizer.
         doc = Doc(self.vocab)
 
-        # The number of characters in the text
-        text_size = len(text)
```

Comment:
I haven't noticed that `text_size` was used multiple times. Can you put it back pleae :'D

Reply:
I kinda already know this was going to happen (:
updated